setup.cfg

```diff
@@ -1,14 +1,14 @@
 [flake8]
+exclude = */migrations/*,.eggs/*
 max-line-length = 120
 max-complexity = 15
-exclude = */migrations/*,.eggs/*
 
 [isort]
 combine_as_imports = True
 enforce_white_space = True
 force_grid_wrap = 4
 include_trailing_comma = True
-known_first_party = cookiecutter_microcosm_sagemaker
+known_first_party = papaya_extractor
 known_standard_library = dataclasses,pkg_resources
 known_third_party = alembic,allennlp,boto3,click,hamcrest,joblib,matplotlib,microcosm,microcosm_sagemaker,networkx,node2vec,nose,numpy,pandas,parameterized,seaborn,six,sklearn,taxonomies,tf,torch,tqdm,unidecode
```

Comment:
I have a longer list if you want to include it

old:
known_third_party = alembic,allennlp,boto3,click,hamcrest,joblib,matplotlib,microcosm,microcosm_sagemaker,networkx,node2vec,nose,numpy,pandas,parameterized,seaborn,six,sklearn,taxonomies,tf,torch,tqdm,unidecode

updated:
known_third_party = alembic,allennlp,boto3,bs4,click,dragnet,hamcrest,joblib,lxml,marshmallow,matplotlib,microcosm,microcosm_flask, microcosm_sagemaker,microcosm_secretsmanager,microcosm_logging,networkx,node2vec,nose,numpy,pandas,parameterized,postal,requests,seaborn,six,sklearn,taxonomies,tf,tldextract,torch,torchtext,tqdm,unidecode,wandb

Reply:
btw, isn't there any way to tell isort that everything is third party unless is first party?